src/assets/scrollbar.css

```
@@ -1,24 +1,24 @@
 
-::-webkit-scrollbar-thumb:hover {
-  background: linear-gradient(var(--neon-magenta), var(--neon-green));
+.cryptotron-app ::-webkit-scrollbar-thumb:hover {
+  background: linear-gradient(var(--cryptotron-neon-magenta), var(--cryptotron-neon-green));
 }
 
 /* Firefox */
-#app-content,
-#app-content :deep(*) {
+.cryptotron-app #app-content,
+.cryptotron-app #app-content :deep(*) {
```

Stylelint (17.11.1)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)